tigerpath/majors_and_certificates/majors/GEO.json

```diff
@@ -5,19 +5,15 @@
   "degree": "AB",
   "urls": [
     "https://ua.princeton.edu/academic-units/department-geosciences",
-    "https://geosciences.princeton.edu/undergraduate/programs-of-study"
+    "https://geosciences.princeton.edu/undergraduate/programs-of-study",
+    "https://geosciences.princeton.edu/sites/geosciences/files/media/information_and_departmental_plan_of_study-revised-02.pdf"
   ],
   "contacts": [
     {
       "type": "Chair",
       "name": "Bess B. Ward",
       "email": "bbw@princeton.edu"
     },
-    {
-      "type": "Associate Chair",
-      "name": "Thomas S. Duffy",

```

Comment:
https://ua.princeton.edu/academic-units/department-geosciences

Reply:
Looks like it's Frederik J. Simons now, can you add that?

Reply:
Updated